flocker/acceptance/test_api.py

```diff
@@ -340,14 +339,16 @@ class DatasetAPITests(TestCase):
     """
     Tests for the dataset API.
     """
-    def test_dataset_creation(self):
+    @require_cluster(1)
+    def test_dataset_creation(self, cluster):
         """
         A dataset can be created on a specific node.
         """
         return create_dataset(self)
 
+    @require_cluster(2)
     @require_moving_backend
-    def test_dataset_move(self):
+    def test_dataset_move(self, cluster):
```

Comment:
Now that `cluster` is passed here, does the test need to start with a call to `get_test_cluster`?  It seems redundant.